infra/base-images/base-builder/detect_repo.py

```diff
@@ -63,15 +74,31 @@ def get_repo(repo_path):
   Returns:
     The repo location or None
   """
-  output, return_code = execute(
-      ['git', 'config', '--get', 'remote.origin.url'],
-      location=repo_path,
-      check_result=True)
+  output, return_code = execute(['git', 'config', '--get', 'remote.origin.url'],
+                                location=repo_path,
+                                check_result=True)
   if return_code == 0 and output:
     return output.rstrip()
   return None
 
 
+def check_for_repo_name(repo_path, repo_name):
+  """Check to see if a github ref exists in a remote repository.
```

Comment:
Is this function checking that repo_name is the name of the remote of the repo located at repo_path.

Reply:
Yes, I will update the comment.